habitat/omnitruck/hooks/init

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "Removing previous version deployed at {{pkg.svc_static_path}}"
+rm -rf {{pkg.svc_static_path}}/* {{pkg.svc_static_path}}/.bundle
+
+echo "Deploying new version from {{pkg.path}} to {{pkg.svc_static_path}}"
+cp -a {{pkg.path}}/static/* {{pkg.path}}/static/.bundle {{pkg.svc_static_path}}
+
+echo "Copying poller config.yml"
+cp -v {{pkg.svc_config_path}}/poller_config.yml {{pkg.svc_static_path}}/config/config.yml
+
+echo "Copying unicorn.rb"
+cp -v {{pkg.svc_config_path}}/unicorn.rb {{pkg.svc_static_path}}/config/unicorn.rb
+
+export GEM_HOME="{{pkg.svc_static_path}}/vendor/bundle/ruby/2.3.0"
+export GEM_PATH="$(hab pkg path core/ruby)/lib/ruby/gems/2.3.0:$(hab pkg path core/bundler):$GEM_HOME"
+export LD_LIBRARY_PATH="$(hab pkg path core/gcc-libs)/lib"
+export SSL_CERT_FILE="$(hab pkg path core/cacerts)/ssl/cert.pem"
+
+echo "Making poller metadata directory"
+mkdir -p {{cfg.poller.metadata_dir}}
+
+echo "Running poller to seed omnitruck data"
+(cd {{pkg.svc_static_path}} && bundle exec ./poller -e production)
```

Comment:
This needs to run on a regular cadence to continue updating the metadata json. We need to find a way to ensure it runs every 5 minutes or so (the old crontab was `*/5 * * * *`).